src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected String color;
+
+    Figure(String color) {

```

Comment:
you might make this constructor `protected` as well, so that subclasses from other packages will also have access to it

no modifier it the same as `package-private` modifier, which in turn means that this constructor is accessible for all the classes within this package, however if you create a subclass of `Figure` in other package - it won't have access to that constructor

```suggestion
    protected Figure(String color) {

```